linera-client/src/chain_listener.rs

```diff
@@ -406,7 +417,9 @@ impl<C: ClientContext> ChainListener<C> {
             .process_inbox_without_prepare()
             .await
         {
-            Err(ChainClientError::CannotFindKeyForChain(_)) => {}
+            Err(ChainClientError::CannotFindKeyForChain(chain_id)) => {
+                debug!(%chain_id, "Cannot find key for chain");
```

Comment:
It might be better to replace this with `err(warn)` instrumentation on `process_inbox_without_prepare`.

Reply:
I originally decided to just ignore this error, because it can happen that one of our chains changes ownership, and then we're not responsible for processing its inbox anymore.

But maybe we should detect it after all, and also remove it from `tracked_chains` in that case.